### DISCUSSION TITLE
My friend got flagged as soon as they joined my repository

### DISCUSSION THREAD

**Discussion by lumbar-puncture-to-the-skull** [community/community#23494] [New to GitHub]
<p>I was working on a game on Unity, and I invited my friend as a pixel artist. They had done literally nothing with their account prior to this, and were completely new to Github. As soon as I sent the invite, they got a notification that their account had been flagged. Neither I nor the repository I invited them to had been flagged. How did this happen?</p>

**Comment by mpboom** [comment, marked as answer]
<p>Hi <a class="mention" href="/dkrazy">@DKrazy</a>, I’m sorry that happened.</p>
<aside class="quote no-group" data-username="DKrazy" data-post="1" data-topic="251778">
<div class="title">
<div class="quote-controls"></div>
 DKrazy:</div>
<blockquote>
<p>How did this happen?</p>
</blockquote>
</aside>
<p>No-one will be able to answer that question, especially on the forums here.</p>
<p>Please get in touch with Support <strong>once</strong>, by following the instructions that are inside your friends’ account. That is the best, fastest and above all only way the account can be unlocked.</p>
<p>On the forums we can never discuss account-specific issues.</p>

**Comment by lumbar-puncture-to-the-skull** [comment]
<p>Hello. I’m sorry if I put this in the wrong place, I’ve never used this platform before.</p>
<p>My friend did contact support, and they replied saying they received a lot of similar messages and it would take them a while to get back to them. My guess is some bug in the website caused a bunch of people to get flagged at once, and Github has to deal with all of it.</p>
<p>I’m probably wrong, but I’m just hoping this can be resolved.</p>

**Comment by ettaboyle** [comment]
<p>No problem <a class="mention" href="/dkrazy">@DKrazy</a> and +1 to <a class="mention" href="/mpboom">@mpboom</a>’s comment. Support is your best bet on account specific questions.</p>
<p>If we see any updates on this topic we will circle back to this thread. Thank you!!</p>
